docs/source/release_notes.rst

```diff
@@ -2,8 +2,18 @@
 
 Release Notes
 -------------
-Future Release
-==============
+.. Future Release
+  ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v1.11.0 June 30, 2022
```

Comment:
June should be Jun, this document always uses first 3 letters of the month

(other releases from this month should also be changed)